docs/history/changelog.md

```diff
@@ -144,7 +144,15 @@ Per the [normative and non-normative content and changes policy](https://docs.go
   * [#1238](https://github.com/open-contracting/standard/pull/1238) Open the `releaseTag` codelist.
   * [#1415](https://github.com/open-contracting/standard/pull/1415) Align descriptions of 'planning', 'tender', 'award' and 'contract' with the corresponding schema fields.
 
-* [#1201](https://github.com/open-contracting/standard/pull/1201) Add 'terminatedEarly' and 'terminatedSuccessfully' codes to the `contractStatus` codelist, to distinguish between successful completion and early termination of the contract.
+* `tenderStatus`, `awardStatus` and `contractStatus`:
+  * [#1509](https://github.com/open-contracting/standard/pull/1509) Major change of descriptions, adding and deprecating the following codes.
+    * Deprecated: 'planning' in `tenderStatus`. Deprecated because it is unnecessary since contracting processes are now split from planning processes (which do not have any states themselves).
+    * Deprecated: 'pending' in `awardStatus` and `contractStatus`. Deprecated because the code (which would follow 'active') does not provide any additional information compared to `awardDate` and `dateSigned` and, in the case of `contractStatus` has unclear semantics ("proposed").
+    * Added: 'withdrawn' in `awardStatus` and `contractStatus`, as information can stop being published at any moment also for only individual awards / contracts.
+    * Added: 'complete' in `awardStatus`, as there was no code suggesting that the award successfully happened and was done (the equivalent in `contractStatus` is 'terminated'.)
+    * Added 'unsuccessful' in `contractStatus`, as a failure not caused by the buyer can happen also in the contract stage (e.g. the winning bidder(s) refuses to sign the contract). 
+  * [#1201](https://github.com/open-contracting/standard/pull/1201) Add 'terminatedEarly' and 'terminatedSuccessfully' to the `contractStatus` codelist, to distinguish between successful completion and early termination of the contract.
+
```

Comment:
Since we haven't yet implemented the replacements for the deprecated fields, we can maybe just leave a simple changelog entry for now, and write something longer when doing the next PR.

Reply:
Done in https://github.com/open-contracting/standard/pull/1509/commits/5fd7ac0a4dc30535f63a80b87c77620afcb597d8. I also added entries for the other changes in this PR.